### ISSUE TITLE
(Read First) Governence of Library

### ISSUE THREAD

**Issue by jimmyhmiller** [trabian/hylomorph#1]
This is the first library like this we really have. If it is going to be successful we have to make sure we figure out how to make decisions and how to cooridinate (or ensure we don't need to coordinate) changes. Initially there a lot of decisions to be made. This issue is about how we will decide those issues. Should we get feedback on an issue and then let someone take that issue and it be their task to take next steps? Should we discuss the issues and assign the initial version to one person?

**Comment by jimmyhmiller** [comment]
Having one person start it out might honestly be the fastest way to get something going when it is this early.

**Comment by trabianmatt** [comment]
@jimmyhmiller agreed - at such an early stage I'd rather see someone with a clear vision for the project (you) take the lead and make decisions, with feedback after the fact if needed.

I think a code of conduct and contributing guidelines would be helpful after we hit a maturity point, thought.
